classes/plugins/PubObjectCache.php

```diff
@@ -43,13 +43,13 @@ public function add($object, $parent)
         }
         if ($object instanceof Submission) {
             $this->_insertInternally($object, 'articles', $object->getId());
-            $this->_insertInternally($object, 'articlesByIssue', $object->getcurrentPublication()->getData('issueId'), $object->getId());
+            $this->_insertInternally($object, 'articlesByIssue', $object->getCurrentPublication()->getData('issueId'), $object->getId());
         }
         if ($object instanceof Galley) {
             assert($parent instanceof Submission);
             $this->_insertInternally($object, 'galleys', $object->getId());
             $this->_insertInternally($object, 'galleysByArticle', $object->getData('submissionId'), $object->getId());
-            $this->_insertInternally($object, 'galleysByIssue', $parent->getIssueId(), $object->getId());
+            $this->_insertInternally($object, 'galleysByIssue', $parent->getCurrentPublication()->getData('issueId'), $object->getId());
```

Comment:
@asmecher Here I just trusted the `assert()` above, I didn't investigate how this code is being used.

Reply:
That looks OK, @jonasraoni, but interesting that it made it through RC1 testing without a fatal error on missing `getIssueId` function being raised. @ewhanson, is the `PubObjectCache` being used in the CrossRef code still?

Reply:
It's not used in the Crossref code, but it looks like the cache is still used in the Datacite plugin, but only as it relates to galleys.

https://github.com/pkp/ojs/blob/e3b451eae41d077412ab72da0b90271242b6bdd4/plugins/generic/datacite/DataciteExportPlugin.php#L477-L482

Reply:
Galleys are the case Jonas found the issue with above. That explains why testing didn't turn it up, thanks! Jonas, please go ahead with the change you've proposed.

Reply:
(^ @jonasraoni)